src/python/pants/backend/go/goals/test.py

```diff
@@ -121,8 +139,36 @@ def transform_test_args(args: Sequence[str]) -> tuple[str, ...]:
 async def run_go_tests(
     field_set: GoTestFieldSet, test_subsystem: TestSubsystem, go_test_subsystem: GoTestSubsystem
 ) -> TestResult:
-    maybe_pkg_info = await Get(
-        FallibleFirstPartyPkgInfo, FirstPartyPkgInfoRequest(field_set.address)
+    maybe_pkg_info, wrapped_target, test_sources = await MultiGet(
```

Comment:
test_sources is unused currently. The thought was that maybe we'd want to include the test sources in the sandbox to better simulate what Go tests see when they are run. (When run, the test binary is run in the same directory as the test sources it was built from.)

Reply:
Do Go tests ever run on their own source code, like `.go` and `.s` files? We don't want `go_package` to be abused to hold unrelated resources - that's what `files` and `resources` are for. So doesn't seem like this makes sense.

Reply:
I've never seen any code that would have run over its own source. So my thought above is moot.